Basic/DistributedAuthoritySocialHub/Assets/Settings/Build Profiles/New macOS Profile.asset

```diff
@@ -0,0 +1,51 @@
+%YAML 1.1
```

Comment:
Should this have been added? I'm getting build profiles warnings in the editor now.

Reply:
Removed, was not intentionally added